src/snapred/backend/recipe/algorithm/PixelGroupingParametersCalculationAlgorithm.py

```diff
@@ -44,64 +55,56 @@ def chopIngredients(self, instrumentState: InstrumentState) -> None:
         self.delLOverL = instrumentState.instrumentConfig.delLOverL
         self.L = instrumentState.instrumentConfig.L1 + instrumentState.instrumentConfig.L2
         self.delL = self.delLOverL * self.L
-        self.delTheta = instrumentState.delTh
-        self.grouping_ws_name = "pgpca_grouping_ws"
-        self.grouped_ws_name = "pgpca_grouped_ws"
-        self.resolution_ws_name = "pgpca_resolution_ws"
-        self.partial_resolution_group_ws_name = "pgpca_partial_resolution_group_ws"
+        self.delTheta = instrumentState.instrumentConfig.delThWithGuide
```

Comment:
```suggestion
        self.delTheta = instrumentState.delTh
```
please change this back, needs to switch depending on guide status.